packages/devextreme/js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -525,9 +526,15 @@ const TextEditorMask = TextEditorBase.inherit({
     },
 
     _updateMaskOption: function() {
+        const valueChangeEvent = addNamespace(this.option(this._getValueChangeEventOptionName()), `${this.NAME}ValueChange`);
+        const $input = this._input();
+
         this._updateHiddenElement();
         this._renderMask();
         this._validateMask();
+        // reattach events to get correct handlers order (T1214019)
+        eventsEngine.off($input, valueChangeEvent);
+        eventsEngine.on($input, valueChangeEvent, this._valueChangeEventHandler.bind(this));
```

Comment:
Cannot we just call already written methods instead of writing an imperative code?
For example, call `_refreshValueChangeEvent`